src/services/session-manager.ts

```diff
@@ -71,7 +71,11 @@ export class OnlineSessionManager implements SessionManager {
         } as AppSession, // Cast required because TS cannot resolve datahubCustomerId complexity
       );
     } catch (error) {
-      this.updateSession({ status: 'non-authenticated' });
+      if (!!error.response && (error.response.status === 503 || error.response.status === 404)) {
```

Comment:
Maybe we can use something like: 
`if ( error?.response.status === 503 || error?.response.status === 404)`
I'm not sure if this works fine.

Reply:
I've tried `error.response?.status` without success 😕

Reply:
Yes, it seems to be that is in that way.

Reply:
![image](https://user-images.githubusercontent.com/2439363/90050590-cbce4500-dcac-11ea-9397-6a244394c1c3.png)
🤔

Reply:
🤷‍♀️